Java/LocalDateTime.md

```diff
@@ -12,9 +12,7 @@ Camel’s case in java programming consists of compound words or phrases such th
 ## Methods in LocalDateTime and Naming Conventions
 | Method Name | Description and Naming Convention |
 |---|---|
-
-
-| String format(DateTimeFormatter formatter)	| It is used to format this date-time using the specified formatter.|
+| String format(DateTimeFormatter formatter)	| It is used to format this date-time using the specified formatter.
```

Comment:
Good that you have gone through the LocalDateTime package, description and methods seems good.
Would be appreciated if written in your own language for better understanding. 
Also when you write maintain a proper structure to write.